src/users/UserItem.js

```diff
@@ -36,7 +36,10 @@ type Props = $ReadOnly<{|
   onPress: UserOrBot => void,
 |}>;
 
-// See UserAvatarWithPresence for discussion of this inexact object type.
+// The underlying class gets an inexact Props type to express that it's fine
+// for it to be passed extra props by the implementation of …ById.
```

Comment:
IIUC, it's not clear what "…ById" refers to in this comment's new home, here, since 912b92a73e8e286711bc1e32b5afd7dc605fd43f.

Reply:
Hmm true! In fact... there's no need for this inexact type in the first place here on `UserItem`, because its Redux-using wrapper is already using Hooks and therefore can pass exactly the desired set of props. So I think I'll just fix that in a prep commit.